### ISSUE TITLE
Shell Bar aria-label not translated

### ISSUE THREAD

**Issue by raman-kalita-sap** [SAP/luigi#4851]
**Description**

The Shell Bar element has an `aria-label` that isn’t translated.

<img width="310" height="79" alt="Image" src="https://github.com/user-attachments/assets/1144c495-31f5-4f7f-b1a4-ac919a6bf396" />  

```.html
<div class="fd-shellbar__branding" role="button" tabindex="0" aria-label="*** i18n-translation-key ***">
  <span class="fd-shellbar__logo  lui-customlogo svelte-8bowjf" aria-hidden="true" tabindex="-1">
    <img data-testid="luigi-topnav-logo" alt="Luigi Fiddle" class="svelte-8bowjf" src="https://fiddle.luigi-project.io/img/luigi.svg">
  </span>
  <span class="fd-shellbar__title lui-shellbar-single-app-title svelte-8bowjf" data-testid="luigi-topnav-title" aria-hidden="true" tabindex="-1">
    *** i18n-translation-value ***
  </span>
</div>
```
Web browser: Chrome Version 144.0.7559.133 (Official Build) (64-bit)
Operating System: Windows 11 Enterprise, 25H2
Screen Reader:  JAWS 2026 version 2026. 2512.50

Violated Accessibility Criteria:  
[WCAG 2.2, 4.1.2, Level A, Name, Role, Value](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html)
[EN 301 549 V3.2.1 (2021-03)](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf) 9.4.1.2, 10.4.1.2, and 11.4.1.2

**Given**
_i18n.json_
```.json
{ "AppName" : "Luigi Fiddle"}
```
_config_
```.js
{
  settings: {
    header: {
      title: 'AppName'
    }
  }
}
```
**Expected result**

Screen Readers (e.g. JAWS) on accessing the button ("Luigi Fiddle" on screenshot) on the header banner region, its label is incorrectly read as "Luigi Fiddle"
```.html
<div . . . aria-label="Luigi Fiddle">
  <span . . . >
    <img . . . >
  </span>
  <span . . . >
    Luigi Fiddle
  </span>
</div>
```

**Actual result**

Screen Readers (e.g. JAWS) on accessing the button ("Luigi Fiddle" on screenshot) on the header banner region, its label is incorrectly read as "AppName"
```.html
<div . . . aria-label="AppName">
  <span . . . >
    <img . . . >
  </span>
  <span . . . >
    Luigi Fiddle
  </span>
</div>
```

**Steps to reproduce**
1. Set up i18n file
```.json
{ "AppName" : "Luigi Fiddle"}
```
2. Set the config
```.js
{
  settings: {
    header: {
      title: 'AppName'
    }
  }
}
```
3. Run an application
4. Open browser's dev tool, find the shell bar, validate the aria-label and the text node.
5. Use a screen reader to observe the page.

**Troubleshooting**

Here:
https://github.com/luigi-project/luigi/blob/8da7fc31c172bb1acd4300612b1e517f76d185c3/core/src/navigation/LogoTitle.svelte#L168  
The `title` is set to `aria-label` directly.

Whereas here:
https://github.com/luigi-project/luigi/blob/8da7fc31c172bb1acd4300612b1e517f76d185c3/core/src/navigation/LogoTitle.svelte#L212 https://github.com/luigi-project/luigi/blob/8da7fc31c172bb1acd4300612b1e517f76d185c3/core/src/navigation/LogoTitle.svelte#L212  
The `title` is set to the node with `$getTranslation`
